pkg/storageincentives/events.go

```diff
@@ -17,6 +17,9 @@ const (
 	claim
 	sample
 	sampleEnd
+	winner
+	frozen
+	idle
```

Comment:
is this used anywhere?

Reply:
`idle` should be the initial state but let me add it. Haven't initiated anywhere.